src/journal/bsod.c

```diff
@@ -0,0 +1,195 @@
+/* SPDX-License-Identifier: LPGL-2.1-or-later */
+
+#include <errno.h>
+#include <getopt.h>
+#include <linux/vt.h>
+#include <sys/ioctl.h>
+#include <stdio.h>
+
+#include "alloc-util.h"
+#include "build.h"
+#include "fd-util.h"
+#include "io-util.h"
+#include "log.h"
+#include "main-func.h"
+#include "sd-id128.h"
+#include "sd-journal.h"
+#include "terminal-util.h"
+
+static void help(void) {
+        printf("%s\n\n"
+               "filters the journal to fetch the first message from the current boot with an emergency log level.\n\n"
```

Comment:
this needs to be line broken. one of the tests in the testuite is tripping up on this, because we want that help texts are 79ch wide at max to deal with traditional terminals which used to be 80x25 wide.

Reply:
Ohhh....
That's interesting.

Thank you sir.




Let me fix and push.